src/monitor/node_metadata.c

```diff
@@ -1521,6 +1521,7 @@ CanInitiateFailover(ReplicationState state)
 {
 	return state == REPLICATION_STATE_SINGLE ||
 		   state == REPLICATION_STATE_PRIMARY ||
+		   state == REPLICATION_STATE_WAIT_PRIMARY ||
```

Comment:
Probably not needed anymore after the discussed wait-primary changes.

Reply:
That right, fixed.